Source/DafnyLanguageServer.Test/GutterStatus/SimpleLinearVerificationGutterStatusTester.cs

```diff
@@ -14,6 +14,24 @@ public class SimpleLinearVerificationGutterStatusTester : LinearVerificationGutt
   // the test will fail and give the correct output that can be use for the test
   // Add '//Next<n>:' to edit a line multiple times
 
+  [TestMethod]
+  public async Task GitIssue3329ModuleRefinementBreaksOutlineErrorReporting() {
+    await VerifyTrace(@"
+ | :abstract module Test {
+ | :  method Encrypt()
+ | :    ensures false
+ | :}
+ | :
+ | :module Ok refines Test {
+ | :  function test(): int {
+ | :    2
+ | :  }
+ | :  // Might not work as expected
+[=]:  method Encrypt() {
+[ ]:  }
+   :}", intermediates: false);
```

Comment:
Previously, this test was entirely green, and only fixing the refinement token would put the postcondition error on line 22, which should not be the case. The real error is now a nested one.